bears/python/Flake8Bear.py

```diff
@@ -0,0 +1,77 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.bears.requirements.PipRequirement import PipRequirement
+from coalib.settings.Setting import typed_list
+
+
+@linter(executable='flake8',
+        output_format='regex',
+        output_regex=r'(?P<filename>.*):(?P<line>\d+):(?P<column>\d+):\s'
+                     r'(?P<severity>[EFW]|C9|N8)[0-9]{2,3}\s'
```

Comment:
`[EFW]|C9|N8)` should be `[A-Z]` - there are lots of flake8 plugins, with different prefixes ; if they have plugins installed, flake8 uses them, and their output should be included at INFO level.